site/schemas/login.schema.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+import { setZodDefaultError } from "../utils";
+
+export const loginSchema = z.object({
+    email: z.string(setZodDefaultError("Enter an email address")).min(1),
```

Comment:
We can use the zod email validation here
```suggestion
    email: z.string().email({ message: "Enter a valid email address" }),
```

Reply:
done